### PR TITLE
update metallb yaml

### PR DESCRIPTION
**Pull request by my-git9**
Ref https://github.com/istio/istio/pull/43049

### CODE REVIEW

File: files/common/scripts/metallb.yaml [hunk 1]
```diff
@@ -374,26 +443,46 @@ spec:
...
             - --config=config
-          image: metallb/controller:v0.9.3
-          imagePullPolicy: Always
+            - --log-level=info
+          env:
+            - name: METALLB_ML_SECRET_NAME
+              value: memberlist
+            - name: METALLB_DEPLOYMENT
+              value: controller
+          image: quay.io/metallb/controller:v0.12.1
```
Comment (howardjohn) [inline files/common/scripts/metallb.yaml:452]:
```suggestion
          image: gcr.io/istio-testing/metallb/controller:v0.12.1
```

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
@jacob-delgado Unfortunately this change breaks the 1.16-1.19 testing that is happening in the earlier releases similar to the lat PR we merged and had to revert in master.

**Pull request merged**